students/7/main1.js

```diff
@@ -0,0 +1,13 @@
+const ex5 = () => {
```

Comment:
Is there a reason why this is in another file?  Can we move this code to main.js and keep everything in one file?

Reply:
it's been deleted!